### ISSUE TITLE
Update to Alfred 4

### ISSUE THREAD

**Issue by Donkeyfish87** [blacs30/bitwarden-alfred-workflow#10]
This workflow throws errors for me now, and I suspect the only problem is that it references to `Alfred 3` a bunch of places.

I also suspect a simple search and replace with Alfred 4 will solve the problems.

I might get a PR  up on this later if I have the time.

**Comment by Donkeyfish87** [comment]
Perhaps this requires updating Deanishe's python stuff to the most recent version as well.

**Comment by blacs30** [comment]
Thanks for reporting this. Do you have a screenshot or the text message itself which occurs to you.
I currently don't have any issues with Alfred 4 but I did upgrade from 3 and might still have some Alfred 3 folders in `~/Library`

**Comment by Donkeyfish87** [comment]
```
21:17:02 workflow.py:2074 ERROR    Command '['/usr/bin/osascript', '-l', 'AppleScript', '-e', '\ntell application "Alfred 3"\nset configuration "STATUS_MESSAGE" to value "Error Login Config" in workflow "com.lisowski-development.alfred.bitwarden" exportable false\nend tell\n']' returned non-zero exit status 1
Traceback (most recent call last):
  File "workflow/workflow.py", line 2067, in run
    func(self)
  File "<string>", line 124, in main
  File "<string>", line 93, in set_notification
  File "workflow/util.py", line 272, in set_config
    run_applescript(script)
  File "workflow/util.py", line 195, in run_applescript
    return run_command(cmd)
  File "workflow/util.py", line 165, in run_command
    return subprocess.check_output(cmd, **kwargs)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 573, in check_output
    raise CalledProcessError(retcode, cmd, output=output)
CalledProcessError: Command '['/usr/bin/osascript', '-l', 'AppleScript', '-e', '\ntell application "Alfred 3"\nset configuration "STATUS_MESSAGE" to value "Error Login Config" in workflow "com.lisowski-development.alfred.bitwarden" exportable false\nend tell\n']' returned non-zero exit status 1
21:17:02 workflow.py:2097 DEBUG    ---------- finished in 0.232s ----------
```

**Comment by blacs30** [comment]
Thanks for the error and the explanation. 
There are a few hard coded `Alfred 3` occurances. Upgrading it directly to Alfred 4 would be possible. Better would be of course to still support Alfred 3 as well.

**Comment by blacs30** [comment]
@Donkeyfish87 I've released version [1.3.0](https://github.com/blacs30/bitwarden-alfred-workflow/releases/tag/1.3.0) which also contains an updated version of the python workflow library to support Alfred 4 now.
